app/components/concept-page/question-card.ts

```diff
@@ -55,7 +63,37 @@ export default class QuestionCardComponent extends Component<Signature> {
   }
 
   @action
-  handleOptionClick(optionIndex: number) {
+  handleMoveDown() {
+    const currentFocusedOption = document.activeElement as HTMLElement;
```

Comment:
```suggestion
    const currentFocusedOption = document.activeElement;
```

It's wrong to assume that there's always a currently focused option - this will not be true.

Reply:
(CodeRabbit highlighted this below)